nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-test/src/test/java/org/nuxeo/directory/test/AbstractDirectoryTest.java

```diff
@@ -868,4 +870,17 @@ public void testPasswordNotReturned() throws Exception {
         }
     }
 
+    @Test
+    public void shouldFailWhenCreateDuplicateEntries() {
+        try {
+            Session session = directoryService.open("continentDirectory");
+            Map<String, Object> continent = Map.of("id", "middle-earth", "label", "Middle Earth");
+            List<Map<String, Object>> entries = Arrays.asList(continent, continent);
+            entries.forEach(session::createEntry);
+            fail();
```

Comment:
You could add a failure message here, something like "Should not be able to create duplicate entries in a dierctory".

Reply:
I will use a simple message like _Should raise a DirectoryException_. The Test method name already hold the information of the Test in the case when it fails.